cicd/scripts/check_license.py

```diff
@@ -35,14 +36,16 @@
 )
 
 
-def check_notice() -> None:
+def check_notice(notice_template: PathLike) -> None:
     """Checks if the NOTICE file is correct and the copyright year is up-to-date.
 
+    Args:
+        notice_template: Path to notice template file.
+
     Raises:
         RuntimeError: If the NOTICE file has incorrect format or the copyright year is not correct.
 
     """
-    notice_template = _TEMPLATES_DIR / "notice.tpl"
     notice_file = _ROOT_PATH / "NOTICE"
```

Comment:
python is not for prepping :/

Reply:
What do you mean by prepping?

Reply:
I need to disable automatic spelling correction in mac os.
* grepping